docs/guides/software-lifecycle/metrics/metrics-starter-kit/install_devlake.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
I would suggest to add License header here.

Reply:
Yes @ddalton-jpl sorry I should have provided an example. A really nice thing about shipping a `LICENSE` file with your source code, and a _really_ nice thing about shipping the Apache Licence v2 is that you are [provided with an example at the tail of the file](https://github.com/NASA-AMMOS/slim/blob/main/LICENSE#L206-L216).
For convenience, here it is :)

```
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#    http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```

I realize that SLIM doesn’t really ship much code… but as you are introducing bash code, I would strongly advise that you add a license header.